docs/Architecture Decision Record/016-dependabot-dependency-management.md

```diff
@@ -35,6 +34,15 @@ To maintain our gitflow:
 
 * (minor) The GitHub README status badges currently don't work for the GitHub Native Dependabot: [as noted in this open dependabot issue](https://github.com/dependabot/dependabot-core/issues/1912)
 
+## Development Dependencies
+This project defines a number of "Development Dependencies" which are not installed in production-esque environments (ie. deployed in Cloud.gov) but are instead only used for supporting operations during local development and automated unit testing in CI/CD.
+
+This dependencies will still be kept up to date by Dependabot but needn't follow the same review flow as updates to packages we actively use within the codebase outside of development/testing. In those cases, as long as the tests all pass on the branch opened to update the dev dependency it may be merged directly into `raft-tdp-main`.
```

Comment:
```suggestion
These dependencies will still be kept up to date by Dependabot but needn't follow the same review flow as updates to packages we actively use within the codebase outside of development/testing. In those cases, as long as the tests all pass on the branch opened to update the dev dependency it may be merged directly into `raft-tdp-main`.
```